src/core/imap/MCIMAPSession.cpp

```diff
@@ -316,6 +316,7 @@ void IMAPSession::init()
     mConnectionType = ConnectionTypeClear;
     mCheckCertificateEnabled = true;
     mVoIPEnabled = true;
+    mNetEaseWorkaroundEnabled = false;
```

Comment:
Yes, The 163.com, 126.com, yeah.net are the mail service provided by the NetEase. All above are used Coremail mail server as the backend.

telnet 126.com
zhanlin@linzhandeMacBook-Air:~$ telnet imap.126.com 143
Trying 220.181.15.127...
Connected to imap.126.com.
Escape character is '^]'.
* OK Coremail System IMap Server Ready(126com[75c606d72bf436dfbce6e08e565f41f7])

telnet yeah.net
zhanlin@linzhandeMacBook-Air:~$ telnet imap.yeah.net 143
Trying 123.58.177.136...
Connected to pop.yeah.split.netease.com.
Escape character is '^]'.
* OK Coremail System IMap Server Ready(yeah[75d5e386074cb656a0bb3fc137855fcf])

telnet 163.com

zhanlin@linzhandeMacBook-Air:~$ telnet imap.163.com 143
Trying 220.181.12.100...
Connected to imap.163.com.
Escape character is '^]'.
* OK Coremail System IMap Server Ready(163com[726cd87d72d896a1ac393507346040fa])